cx_Freeze/freezer.py

```diff
@@ -610,6 +610,8 @@ def _write_modules(self, filename: Path, finder: ModuleFinder):
                             parts.append("__init__")
                         target_name = target_lib_dir.joinpath(*parts)
                         target_name = target_name.with_suffix(".pyc")
+                        if not os.path.exists(target_name.parent):
```

Comment:
Please, change to:
self._create_directory(target_name.parent)

Reply:
Done

Reply:
You can remove the if. I mean, change the two lines to one.